perceval/client.py

```diff
@@ -236,6 +236,8 @@ def sleep_for_rate_limit(self):
         """
         if self.rate_limit is not None and self.rate_limit <= self.min_rate_to_sleep:
             seconds_to_reset = self.calculate_time_to_reset()
+            seconds_to_reset = 1 if seconds_to_reset < 0 else seconds_to_reset
```

Comment:
If seconds to rest is zero or less than zero why to sleep one second? It should sleep 0.

Reply:
you are right, it should be 0